src/index.ts

```diff
@@ -83,14 +83,23 @@ directLine.activity$
   // .filter(activity => activity.type === "message")
   .subscribe((message: DirectLineMessage) => {
     logger.log("received message ", message);
-    let lineMessage:Line.Message;
-    if(message.attachments != null && message.attachments[0].contentType == "application/vnd.microsoft.card.video"){
-      const videoConverter = new VideoConverter();
-      lineMessage = videoConverter.DirectLineToLine(message);
-    }else
-    {
+    let lineMessage: Line.Message;
+    if (message.attachments != null && message.attachments[0].contentType != null) {
+      switch (message.attachments[0].contentType) {
```

Comment:
I think `attachments` can have multiple types of content. So it's better to use `for` or `forEach` and check the type each of content and pass it to the converter.